src/Servers/Kestrel/Core/test/Http1ConnectionTests.cs

```diff
@@ -127,9 +127,7 @@ public async Task TakeMessageHeadersThrowsWhenHeadersExceedTotalSizeLimit()
             await _application.Output.WriteAsync(Encoding.ASCII.GetBytes($"{headerLine}\r\n"));
             var readableBuffer = (await _transport.Input.ReadAsync()).Buffer;
 
-#pragma warning disable CS0618 // Type or member is obsolete
-            var exception = Assert.Throws<BadHttpRequestException>(() => TakeMessageHeaders(readableBuffer, trailers: false, out _consumed, out _examined));
-#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
Please tell me you kept at least some of the old pragmas? We need something that verifies the old exception is still the one thrown.

Reply:
There's still one pragma in this test class because it's checking some internal property. I didn't even touch the other test classes.